microservices-connector/api/v1alpha3/validating_webhook.go

```diff
@@ -34,6 +34,7 @@ var (
 		"TeiReranking",
 		"Tgi",
 		"TgiGaudi",
+		"TgiNvidia",
```

Comment:
is this mandatory?

Reply:
Yes, otherwise the CR `config/samples/chatQnA_nv.yaml` would fail for invalid stepName error.